Cargo.toml

```diff
@@ -43,13 +44,13 @@ wgsl-in = ["codespan-reporting"]
 wgsl-out = []
 hlsl-out = []
 
-[[bin]]
-name = "naga"
-path = "bin/naga.rs"
-
 [dev-dependencies]
 diff = "0.1"
 ron = "0.6"
 serde = { version = "1.0", features = ["derive"] }
 spirv = { package = "spirv_headers", version = "1.5", features = ["deserialize"] }
 rspirv = "0.7"
+
+[workspace]
+members = [".", "naga-cli"]
```

Comment:
Can this live in `cli` folder? We have `naga-fuzz` today sitting there under just `fuzz`, so this would be consistent.
I don't recall why the fuzzer needs its own workspace tho